docs/source/install.rst

```diff
@@ -100,7 +100,7 @@ Here is an example of how to create and use the AWS SDK for pandas Lambda layer
           "awssdkpandas-layer",
           location=sam.CfnApplication.ApplicationLocationProperty(
             application_id="arn:aws:serverlessrepo:us-east-1:336392948345:applications/aws-sdk-pandas-layer-py3-8",
-            semantic_version="2.20.0",  # Get the latest version from https://github.com/aws/aws-sdk-pandas/releases
+            semantic_version="2.20.1",  # Get the latest version from https://github.com/aws/aws-sdk-pandas/releases
```

Comment:
I don't think we will be creating a new SAR for a minor version so would bring it back to 2.20 perhaps?

Reply:
As discussed on Slack, we'll still publish layers for this patch. Although it's a very small patch, the issue its addressing can have a potentially large customer impact.